### PR TITLE
Details Popup window

### PR DESCRIPTION
**Pull request by wajahatalli**
- Added a modal using javascript.
- Created an object to store content to be fed to the main page.
- Fixed eslinter error
- Fixed image position
- Fixed text position in the mobile version
- Made all required changes
- Its perfect now

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    technologies: ['html', 'css', 'JavaScript'],
+    link: 'https://github.com/wajahatalli/Portfolio/',
+    source: 'https://github.com/wajahatalli/Portfolio',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    description:
+            'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the1500, when an unknown printer took a galley of type and scrambled it 1960s.',
+    image: '../images/work-img2.png',
```
Comment (tufoinnkuo10) [inline index.js:33]: - [ ] kindly ensure that in the mobile version,  the text is aligned with the image and technologies like in the figma version
Screenshot (593)_LI

File: index.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    technologies: ['html', 'css', 'JavaScript', 'github'],
+    link: 'https://github.com/wajahatalli/Portfolio/',
+    source: 'https://github.com/wajahatalli/Portfolio/',
+  },
+  {
+    id: 3,
+    name: 'Facebook 360',
+    description:
```
Comment (tufoinnkuo10) [inline index.js:41]: - [ ] please centralize your image in the desktop popup , align it with the text and see source button like in the figma design. great work.
yours
Screenshot (594)_LI
figma
Screenshot (595)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @wajahatalli 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by wajahatalli** [comment]
- Fixed eslinter error
- Fixed image position
- Fixed text position in the mobile version
- Made all required changes
- Its perfect now

**Review by PinkMoon25 — APPROVED** [review]
Hi @wajahatalli , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 You used the correct Github flow :heavy_check_mark: 
 Descriptive pull request message :heavy_check_mark: 
 linter error checks validated :heavy_check_mark: 
Documented your work professionally :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**